src/cloud-api-adaptor/podvm-mkosi/Makefile

```diff
@@ -132,6 +137,7 @@ image-container:
 	docker buildx build \
 		-t $(PODVM_CONTAINER_NAME):$(PODVM_TAG) \
 		-t $(PODVM_CONTAINER_NAME):latest \
```

Comment:
Maybe remove `-t $(PODVM_CONTAINER_NAME):latest` for consistency.

Reply:
Consistency with what? The other mkosi generated image have both the `podvm` tag and `latest` tag?